src/polysolve/LinearSolverSympiler.hpp

```diff
@@ -0,0 +1,59 @@
+#ifdef POLYSOLVE_WITH_SYMPILER
+
+#include <polysolve/LinearSolver.hpp>
+#include <sympiler/parsy/cholesky_solver.h>
+#include <Eigen/Sparse>
+#include <Eigen/Core>
+
+namespace polysolve
+{
+    class LinearSolverSympiler : public LinearSolver
+    {
+    public:
+        LinearSolverSympiler() = default;
+        ~LinearSolverSympiler() = default;
```

Comment:
It is a good idea to either mark the class as `final` or make sure to use `virtual` destructor.

Reply:
Fixed